Catappult/appcoins-wallet-android
# [APPC-3229] Vip Referral codes

**What does this PR do?**
New screen to share Vip referral codes automatically. New button and logic on the promotions page to show the Vip referral if active.

**Database changed?**
No

**How should this be manually tested?**
Use a recent vip wallet and check the promotions screen and vip referral screen.

**What are the relevant tickets?**
https://aptoide.atlassian.net/browse/APPC-3229

**Code Review Checklist**
